controllers/ratelimitpolicy_controller_test.go

```diff
@@ -64,7 +64,7 @@ func testBuildBasicHttpRoute(routeName, gwName, ns string) *gatewayapiv1alpha2.H
 		},
 		Spec: gatewayapiv1alpha2.HTTPRouteSpec{
 			CommonRouteSpec: gatewayapiv1alpha2.CommonRouteSpec{
-				ParentRefs: []gatewayapiv1alpha2.ParentRef{
```

Comment:
wow, this is unexpected. They broke the API from 0.4 to 0.5. I guess this kind of things happens when working with 0.X versions.

Reply:
Exactly this, I was a bit surprised too... but then, well, same conclusion.